docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -310,20 +316,39 @@ $ cp -R /data/wordpress-pvc/* .
 [start=3]
 . Since it is a PHP application, nothing else is needed for this pod.
 
-In the case of the MariaDB pod, the application must be restarted after restoring the files:
+=== Restoring the database dump
+
+In the case of the MariaDB pod, this backup annotation in the MariaDB deployment instructed K8up to create a database dump as text and store it in MinIO:
+
+[source,yaml]
+....
+spec:
+  template:
+    metadata:
+      annotations:
+        k8up.syn.tools/backupcommand: appuio.ch/backupcommand: /bin/bash -c 'mysqldump -uroot -p"${MARIADB_ROOT_PASSWORD}" --all-databases'
+....
+
+Thus the restoration procedure is different. 
+Instead of restoring files to a PVC, we can instead connect to the running database in the pod and restore directly from the dump.
 
 [start=4]
-. Follow the same logic as previously:
+. Get the snapshot ID for the database dump and save it to your local machine as `backup.sql`:
 
 ....
-$ cd /var/lib/mysql # Attention! Different path!
-$ mv data /
-$ rm -r *
-$ cp -R /data/mariadb-pvc/* .
+$ export SNAPSHOT_ID=$(restic snapshots --json --last --path /default-mariadb | jq -r '.[0].id')
+$ restic dump "${SNAPSHOT_ID}" /default-mariadb > backup.sql
 ....
 
 [start=5]
-. Delete the MariaDB pod and a new one will be immediately created due to the deployment policy (see `mariadb/deployment.yml`): `kubectl delete pod mariadb-XXXXXXXXXX-xxxx`
+. Now connect to the MariaDB pod, copy the dump and a simple restore script to the pod and execute it there:
+
+....
+$ export MARIADB_POD=$(kubectl get pods | grep mariadb | awk '{print $1}')
+$ kubectl cp backup.sql "$MARIADB_POD":/
+$ kubectl cp scripts/db_restore.sh "$MARIADB_POD":/
+$ kubectl exec "$MARIADB_POD" -- /db_restore.sh
```

Comment:
Isn't there are more direct way to get the pods? e.g.
`kubectl get pods -o custom-columns="NAME:.metadata.name" --no-headers -l app.kubernetes.io/name=mariadb`

or probably even `kubectl exec deploy/mariadb -- /db_restore.sh` (s/deploy/sts/ if statefulset) so you won't have to select a specific pod first.

Reply:
I took those lines pretty much verbatim from `scripts/4_restore.sh` but I see what you mean. I've adapted to: `kubectl get pods -o custom-columns="NAME:.metadata.name" --no-headers -l "app=wordpress,tier=mariadb"` since the  `-l app.kubernetes.io/name=mariadb` didn't seem to hit that pod (?)

`kubectl exec deploy/mariadb -- /db_restore.sh` does work but now that `$MARIADB_POD` is set, wouldn't it be OK to use it in all three lines?

Reply:
yeah this `-l app.kubernetes.io/name=mariadb` was just prosa guessing in the blue. I didn't actually know which labels ;)
ah I overlooked the kubectl cp part. `kubectl cp` doesn't seem to support the deploy/ syntax, at least not in `--help`.
sure, you could keep `kubectl exec $MARIADB_POD` for consistency, no hard feeling here.

Reply:
Ah, alright :)